clients/linux/src/app.rs

```diff
@@ -1079,6 +1121,79 @@ impl LbApp {
         Ok(())
     }
 
+    fn paste_in_text_area(&self, info: TextAreaDropPasteInfo) -> LbResult<()> {
+        let mark = self.gui.account.get_cursor_mark()?;
+
+        let opened_file = self
+            .state
+            .borrow()
+            .opened_file
+            .clone()
+            .ok_or_else(|| uerr_dialog!("Open a file before pasting!"))?;
+
+        match info {
+            TextAreaDropPasteInfo::Image(bytes) => {
+                let parent_path = self.core.full_path_for(&opened_file.parent)?;
+
+                let gdk_win = self.gui.account.cntr.get_window().unwrap();
+                gdk_win.set_cursor(Cursor::from_name(&gdk_win.get_display(), "wait").as_ref());
+
+                let image_name = format!("img-{}.{}", Uuid::new_v4(), "jpeg");
```

Comment:
How are you able to assume jpeg?

Reply:
Apart from ☝️ I'm good with this when @steverusso is good with this.

In the future we'll probably want a progress indicator, but in release mode for huge files (100MB RAW) this is already good enough for now, IMO.

Reply:
It isn't so much as assuming as it is that pixbuf doesn't expose a way to view the file format of the image it contains. If something is of type pixbuf (like what I get from the clipboard or drop), then it is able to be changed into any of the wide array of file formats it supports. Probably adding a dialog to choose would be good. Another way I could go about this is to get the raw bytes from the clipboard and drop, which should in turn preserve its type, but I would likely need to import a library or something to find out the file format to have the correct extension for the file created from it.

Reply:
https://valadoc.org/gdk-pixbuf-2.0/Gdk.Pixbuf.save_to_bufferv.html

Gotcha, okay can we save as png, if the image has an alpha layer, then jpeg will blast it away

Reply:
Oh, I didn't know it didn't support alpha. Gotcha, will save it as png.